README.md

```diff
@@ -153,6 +153,10 @@ in the Installation Guide.
 
 2.  Clone the sources with `git`:
 
+    - [Windows] **The Sage source tree contains symbolic links, which use UNIX line endings.
```

Comment:
"symbolic links" and "UNIX line endings" are two separate, unrelated things

Reply:
Maybe it should be clarified that it's using Windows line endings that ends up breaking the symlinks?